cli/jobs/automl-standalone-jobs/cli-automl-image-classification-multiclass-task-fridge-items/How to Run this AutoML Job with CLI (Image Classif multi-class).txt

```diff
@@ -0,0 +1,25 @@
+
+How to Run this AutoML Job with CLI (Image Classification multi-class)
+===============================================================
+
+
+1. Prepare data/images (Upload images to Workspace and generate JSONL dataset)
+------------------------------------------------------------------------------
+
+Because the image based datasets need to be uploaded and second the JSONL dataset be generated based on the uploaded images URIs, you first need to 
```

Comment:
Should this sentence end with ", you first need to prepare data." ? Same for other .txt files for image tasks

Reply:
Fixed, thanks! :)